geocoding/example/lib/plugin_example/geocode_page.dart

```diff
@@ -75,9 +75,11 @@ class _GeocodeWidgetState extends State<GeocodeWidget> {
 
                     placemarkFromCoordinates(latitude, longitude)
                         .then((placemarks) {
-                      final output = placemarks[0].toString();
+                      var output = 'No results found.';
+                      if (placemarks != null && placemarks.isNotEmpty) {
```

Comment:
Since our last change the `placemarks` variable should never be `null` so we can simplify this check:

```suggestion
                      if (placemarks.isNotEmpty) {
```